### PR TITLE
Kristian components

### PR DESCRIPTION
**Pull request by Frosk-Kristian**

### CODE REVIEW

File: CollisionDetection/CollisionDetection.cpp [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+		a.max.y >= b.min.y && a.min.y <= b.max.y &&
+		a.max.z >= b.min.z && a.min.z <= b.max.z) {
+		return true;
+	} // end if
+
+	return false;
+} // end BoxCollision
+
+bool CollisionDetection::SphereCollision(Sphere &a, Sphere &b) {
+	float distance = sqrtf(Squaref(b.center.x - a.center.x) + Squaref(b.center.y - a.center.y) + Squaref(b.center.z - a.center.z));
```
Comment (crasheds22) [inline CollisionDetection/CollisionDetection.cpp:18]: Each should be flipped: a.centre.__ - b.centre.__

**Comment by crasheds22** [comment]
Youll find this useful: https://developer.mozilla.org/en-US/docs/Games/Techniques/3D_collision_detection

File: CollisionDetection/CollisionDetection.cpp (hunk 1 above)
Comment (Frosk-Kristian) [reply CollisionDetection/CollisionDetection.cpp:18]: Alright, I'll make that change later tonight.

**Comment by crasheds22** [comment]
[Collider Components + Detection functions](https://trello.com/c/xgEw7OVO/40-collider-components-detection-functions)

**Comment by Frosk-Kristian** [comment]
closing the pull request to make a new one with the updated code

**Pull request closed without merging**